### DISCUSSION TITLE
[Hotchocolate v13.0.5] Subscription stopped at 64th message after reconnection

### DISCUSSION THREAD

**Discussion by dave-chow** [ChilliCream/hotchocolate#6094] [General]
I tried to setup a Graphql subscription in v13.0.5, it works normally at first, but failed to get the message in banana playground after reconnection
Look like the connection still healthy but no message is going to publish after hitting the TopicBufferCapacity (64th messages), even the data can be found in Redis PubSub
Here is the way to reproduce: https://ibb.co/RBbM59y

    // Subscription:
	[Subscribe(With = nameof(SubscribeStressTestTask))]
	public InputPayload SubscribeStressTest([EventMessage] InputPayload d)
	{
		return d;
	}

	public async ValueTask<ISourceStream<InputPayload>> SubscribeStressTestTask([Service] ITopicEventReceiver eventReceiver, CancellationToken cancellationToken)
	{
		return await eventReceiver.SubscribeAsync<InputPayload>(OmsSubscriptionTopics.StressTest(), cancellationToken);
	}

    public static class OmsSubscriptionTopics
    {
	    public static string StressTest() => "SubscribeStressTest";
    }

    // Publish:
	public async Task<bool> ManualStressPublish(int input, [Service] ITopicEventSender topicEventSender, CancellationToken cancellationToken)
	{
		var so = new InputPayload();
		so.OrderId = "12345678-c3d9-4b23-81f9-b6d763100ad0";
		so.AddedByUserId = "DAVE TESTING";
		so.InfoMessage = "MANUAL INPUT BY DAVE";
		for (var i = 0; i < input; i++)
		{
			so.Size = i.ToString();
			await topicEventSender.SendAsync(OmsSubscriptionTopics.StressTest(), so, cancellationToken);
		}
		return true;
	}

**Comment by dave-chow** [comment]
It is not happening in Hotchocolate v12
And I have a temp workaround in changing the TopicBufferFullMode to the following, but I do not think it is a fix for it.

return await eventReceiver.SubscribeAsync<InputPayload>(OmsSubscriptionTopics.StressTest(), 1000, TopicBufferFullMode.DropOldest, cancellationToken);

**Comment by tobias-tengler** [comment]
This was already reported in https://github.com/ChilliCream/graphql-platform/issues/5877 and the fix should be coming with the next preview release.

**Reply by dave-chow** [reply]
Thanks so much!
